emissionsapi/download.py

```diff
@@ -28,24 +24,33 @@ def download():
     """Download data files from ESA and store them in the configured storage
     directory.
     """
-    wkt = bounding_box_to_wkt(*country_bounding_boxes['DE'][1])
+
+    # Load download configuration
+    storage = config('storage') or 'data'
+    date_begin = config('download', 'date', 'begin') or '2019-09-10T00:00:00.000Z'
```

Comment:
Could you add a comment why you default to this date?

Reply:
So this seems to be a randomly chosen value, that was chosen at the very beginning of the project, just to have a very small time window to download from.